### ISSUE TITLE
Can't Launch Brackets

### ISSUE THREAD

**Issue by Mixerito** [adobe/brackets#11783]
When running from command line, no error appears.
Cache clearing doesn't help. 

<b>Report.wer :</b>
Version=1
EventType=APPCRASH
EventTime=130886469647424755
ReportType=2
Consent=1
UploadTime=130886469648048756
ReportIdentifier=36e2c3b4-6c80-11e5-84ca-2c41380284fa
IntegratorReportIdentifier=36e2c3b3-6c80-11e5-84ca-2c41380284fa
Response.BucketId=206124077
Response.BucketTable=25
Response.type=4
Sig[0].Name=Název aplikace
Sig[0].Value=Brackets.exe
Sig[1].Name=Verze aplikace
Sig[1].Value=1.4.0.0
Sig[2].Name=Časové razítko aplikace
Sig[2].Value=55b5ceaf
Sig[3].Name=Název chybného modulu
Sig[3].Value=Brackets.exe
Sig[4].Name=Verze chybného modulu
Sig[4].Value=1.4.0.0
Sig[5].Name=Časové razítko chybného modulu
Sig[5].Value=55b5ceaf
Sig[6].Name=Kód výjimky
Sig[6].Value=c0000005
Sig[7].Name=Posun výjimky
Sig[7].Value=0002c968
DynamicSig[1].Name=Verze operačního systému
DynamicSig[1].Value=6.1.7601.2.1.0.256.48
DynamicSig[2].Name=ID národního prostředí
DynamicSig[2].Value=1029
DynamicSig[22].Name=Další informace 1
DynamicSig[22].Value=df36
DynamicSig[23].Name=Další informace 2
DynamicSig[23].Value=df3604580e0e9976104b661d3b86907b
DynamicSig[24].Name=Další informace 3
DynamicSig[24].Value=5fb7
DynamicSig[25].Name=Další informace 4
DynamicSig[25].Value=5fb7da3112a9f9748dc66cd50ba23744
UI[2]=C:\Program Files\Brackets14\Brackets.exe
UI[3]=Program Brackets přestal pracovat.
UI[4]=Systém Windows může vyhledat řešení problému online.
UI[5]=Vyhledat řešení online a ukončit program
UI[6]=Vyhledat řešení online později a ukončit program
UI[7]=Ukončit program
LoadedModule[0]=C:\Program Files\Brackets14\Brackets.exe
LoadedModule[1]=C:\Windows\SYSTEM32\ntdll.dll
LoadedModule[2]=C:\Windows\system32\kernel32.dll
LoadedModule[3]=C:\Windows\system32\KERNELBASE.dll
LoadedModule[4]=C:\Windows\system32\VERSION.dll
LoadedModule[5]=C:\Windows\system32\msvcrt.dll
LoadedModule[6]=C:\Windows\system32\PSAPI.DLL
LoadedModule[7]=C:\Windows\system32\WINMM.dll
LoadedModule[8]=C:\Windows\system32\USER32.dll
LoadedModule[9]=C:\Windows\system32\GDI32.dll
LoadedModule[10]=C:\Windows\system32\LPK.dll
LoadedModule[11]=C:\Windows\system32\USP10.dll
LoadedModule[12]=C:\Windows\system32\SHLWAPI.dll
LoadedModule[13]=C:\Program Files\Brackets14\libcef.dll
LoadedModule[14]=C:\Windows\system32\ADVAPI32.dll
LoadedModule[15]=C:\Windows\SYSTEM32\sechost.dll
LoadedModule[16]=C:\Windows\system32\RPCRT4.dll
LoadedModule[17]=C:\Windows\system32\WINSPOOL.DRV
LoadedModule[18]=C:\Windows\system32\COMDLG32.dll
LoadedModule[19]=C:\Windows\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.7601.18837_none_41e855142bd5705d\COMCTL32.dll
LoadedModule[20]=C:\Windows\system32\SHELL32.dll
LoadedModule[21]=C:\Windows\system32\ole32.dll
LoadedModule[22]=C:\Windows\system32\OLEAUT32.dll
LoadedModule[23]=C:\Windows\system32\WINHTTP.dll
LoadedModule[24]=C:\Windows\system32\webio.dll
LoadedModule[25]=C:\Windows\system32\USERENV.dll
LoadedModule[26]=C:\Windows\system32\profapi.dll
LoadedModule[27]=C:\Windows\system32\urlmon.dll
LoadedModule[28]=C:\Windows\system32\api-ms-win-downlevel-ole32-l1-1-0.dll
LoadedModule[29]=C:\Windows\system32\api-ms-win-downlevel-shlwapi-l1-1-0.dll
LoadedModule[30]=C:\Windows\system32\api-ms-win-downlevel-advapi32-l1-1-0.dll
LoadedModule[31]=C:\Windows\system32\api-ms-win-downlevel-user32-l1-1-0.dll
LoadedModule[32]=C:\Windows\system32\api-ms-win-downlevel-version-l1-1-0.dll
LoadedModule[33]=C:\Windows\system32\api-ms-win-downlevel-normaliz-l1-1-0.dll
LoadedModule[34]=C:\Windows\system32\normaliz.DLL
LoadedModule[35]=C:\Windows\system32\iertutil.dll
LoadedModule[36]=C:\Windows\system32\WININET.dll
LoadedModule[37]=C:\Windows\system32\dhcpcsvc.DLL
LoadedModule[38]=C:\Windows\system32\WS2_32.dll
LoadedModule[39]=C:\Windows\system32\NSI.dll
LoadedModule[40]=C:\Windows\system32\WTSAPI32.dll
LoadedModule[41]=C:\Windows\system32\Secur32.dll
LoadedModule[42]=C:\Windows\system32\SSPICLI.DLL
LoadedModule[43]=C:\Windows\system32\CRYPT32.dll
LoadedModule[44]=C:\Windows\system32\MSASN1.dll
LoadedModule[45]=C:\Windows\system32\IPHLPAPI.DLL
LoadedModule[46]=C:\Windows\system32\WINNSI.DLL
LoadedModule[47]=C:\Windows\system32\IMM32.dll
LoadedModule[48]=C:\Windows\system32\MSCTF.dll
LoadedModule[49]=C:\Windows\system32\OLEACC.dll
LoadedModule[50]=C:\Windows\system32\SETUPAPI.dll
LoadedModule[51]=C:\Windows\system32\CFGMGR32.dll
LoadedModule[52]=C:\Windows\system32\DEVOBJ.dll
LoadedModule[53]=C:\Windows\WinSxS\x86_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.18946_none_72d45ee78666ea32\gdiplus.dll
LoadedModule[54]=C:\Windows\system32\dwmapi.dll
LoadedModule[55]=C:\Windows\system32\CRYPTBASE.dll
LoadedModule[56]=C:\Windows\system32\uxtheme.dll
LoadedModule[57]=C:\Program Files\Stardock\ObjectDockPlus2\DockShellHook.dll
LoadedModule[58]=C:\Program Files\MaxMax\MaxMaxH.dll
LoadedModule[59]=C:\Windows\WinSxS\x86_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.6161_none_50934f2ebcb7eb57\MSVCP90.dll
LoadedModule[60]=C:\Windows\WinSxS\x86_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.6161_none_50934f2ebcb7eb57\MSVCR90.dll
LoadedModule[61]=C:\Windows\system32\NLAapi.dll
LoadedModule[62]=C:\Windows\system32\audioses.dll
LoadedModule[63]=C:\Windows\system32\MMDevAPI.DLL
LoadedModule[64]=C:\Windows\system32\PROPSYS.dll
LoadedModule[65]=C:\Windows\system32\CLBCatQ.DLL
LoadedModule[66]=C:\Windows\system32\dhcpcsvc6.DLL
LoadedModule[67]=C:\Windows\system32\WINTRUST.dll
LoadedModule[68]=C:\Windows\system32\mswsock.dll
LoadedModule[69]=C:\Windows\system32\apphelp.dll
LoadedModule[70]=C:\Windows\System32\wshtcpip.dll
State[0].Key=Transport.DoneStage1
State[0].Value=1
State[1].Key=DataRequest
State[1].Value=Bucket=206124077/nBucketTable=25/nResponse=1/n
FriendlyEventName=Práce zastavena
ConsentKey=APPCRASH
AppName=Brackets
AppPath=C:\Program Files\Brackets14\Brackets.exe

**Comment by abose** [comment]
did you try deleting `%appdata%/brackets` folder?

**Comment by Mixerito** [comment]
Yes, as i mentioned before, i first deleted cache folder (cef_data), as proposed in troubleshooting guide, then extensions folder too and finally the whole Brackets folder. Nothing of these helped.

**Comment by Mixerito** [comment]
There are values from register probably causing this bug. Deleting these helps to solve this issue.
![reg](https://cloud.githubusercontent.com/assets/14989524/10400153/5d79204c-6eb7-11e5-8f92-2ef4238455e6.png)

**Comment by abose** [comment]
tagging @nethip
